prog/vm/nexus.rb

```diff
@@ -33,9 +33,11 @@ def self.assemble(public_key, project_id, name: nil, size: "m5a.2x",
       private_subnets.append([random_ula, random_private_ipv4])
     end
 
+    line, core_count = Vm.parse_vm_size(size)
```

Comment:
I can see why do you split it here but I think it would be better to have the same separation in the assemble api. Should we consider splitting it there as well and have a matrix of supported line: [core_counts]. That way, in the API, we can run the check and validate the request.